flink-table/flink-table-runtime/pom.xml

```diff
@@ -64,35 +62,38 @@ under the License.
 			<version>${project.version}</version>
 		</dependency>
 
+		<!-- Flink dependencies -->
 		<dependency>
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-streaming-java</artifactId>
 			<version>${project.version}</version>
-			<scope>provided</scope>
```

Comment:
Why not `provided` anymore?

Reply:
Same as https://github.com/apache/flink/pull/18134#discussion_r773743200

Reply:
Actually, for this one I can remove it as we pull it from api-java-bridge